dsaas-services/che-devfile-registry.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+- hash: 314caebc5bd0cb8e26360a650b646a047bd8a048
+  hash_length: 7
+  name: che-devfile-registry
+  path: /deploy/openshift/che-devfile-registry.yml
```

Comment:
Upstream filename has a `.yaml` extension

Reply:
Good catch. Sorry for this copy/paste error

Reply:
@jfchevrette fixed